charts/spark-operator-chart/values.yaml

```diff
@@ -156,10 +156,14 @@ controller:
     #   memory: 300Mi
 
   # -- Security context for controller containers.
-  securityContext: {}
-    # runAsUser: 1000
-    # runAsGroup: 2000
-    # fsGroup: 3000
+  securityContext:
+    runAsNonRoot: true
```

Comment:
Could add `allowPrivilegeEscalation: false` as well

Reply:
I did not add this as the default value of `allowPrivilegeEscalation` should be `false`.

Reply:
Done.